# Rviz display plugin type
string class_name

# Topic that is used to send info to rviz plugin
string topic

# Plugin configuration XML string
string config

temoto_resource_registrar/TemotoRequestMetadata temoto_metadata
---

temoto_resource_registrar/TemotoResponseMetadata temoto_metadata
